operator/json/v0/config/definition.json

```diff
@@ -2,6 +2,7 @@
   "available_tasks": [
     "TASK_MARSHAL",
     "TASK_UNMARSHAL",
+    "TASK_UNMARSHAL_LIST",
```

Comment:
Important
I am not sure yet.
But, I think we can do it with iterator under the current design.
Please try it out first.

Reply:
Yes, it could be done with the `Iterator` component, removed this task.